### PR TITLE
fix netperf compile issue when gcc version >= 10

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/tools/gcc.py [hunk 1]
```diff
@@ -22,6 +22,14 @@ def compile(self, filename: str, output_name: str = "") -> None:
         else:
             self.run(filename)
 
+    def version(self) -> int:
```
Comment (squirrelsc) [inline lisa/tools/gcc.py]: change it like "verbose-noun", for example "get_version", because it's a method.

File: lisa/tools/gcc.py [hunk 2]
```diff
@@ -22,6 +22,14 @@ def compile(self, filename: str, output_name: str = "") -> None:
...
+        output = self.run(
+            "-dumpversion",
+            expected_exit_code=0,
+            expected_exit_code_failure_message="fail to get gcc dumpversion",
+        )
+        return int(output.stdout.split(".")[0])
```
Comment (squirrelsc) [inline lisa/tools/gcc.py]: how about return the "VersionInfo" type? Maybe the minor version is needed in future.

File: lisa/tools/gcc.py (hunk 1 above)
Comment (LiliDeng) [reply lisa/tools/gcc.py]: update

File: lisa/tools/gcc.py (hunk 2 above)
Comment (LiliDeng) [reply lisa/tools/gcc.py]: update

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**